presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/util/RetryDriver.java

```diff
@@ -226,4 +263,87 @@ void nextAttempt()
             }
         }
     }
+
+    public static final class RetryStats
+    {
+        private final CounterStat success = new CounterStat();
```

Comment:
Let's put fields and getters in certain order. For example: all success, then all failures, or for each group do success then failure.